privacy-policy.md

```diff
@@ -0,0 +1,210 @@
+ * **To deliver and facilitate delivery of services to the user.** We may process your information to provide you with the requested service.
+ * **To respond to user inquiries/offer support to users.** We may process your information to respond to your inquiries and solve any potential issues you might have with the requested service.
+ * **To send administrative information to you.** We may process your information to send you details about our products and services, changes to our terms and policies, and other similar information.
+ * **To enable user-to-user communications.** We may process your information if you choose to use any of our offerings that allow for communication with another user.
+ * **To save or protect an individual's vital interest.** We may process your information when necessary to save or protect an individual’s vital interest, such as to prevent harm.
+
+
+### 3. What Legal Bases Do We Rely On To Process Your Personal Information?
+
+**In Short:**
+We only process your personal information when we believe it is necessary and we have a valid legal reason (i.e. legal basis) to do so under applicable law, like with your consent, to comply with laws, to provide you with services to enter into or fulfil our contractual obligations, to protect your rights, or to fulfil our legitimate interests.
+
+The General Data Protection Regulation (GDPR) and UK GDPR require us to explain the valid legal bases we rely on in order to process your personal information. As such, we may rely on the following legal bases to process your personal information:
+ * **Consent.** We may process your information if you have given us permission (i.e. consent) to use your personal information for a specific purpose. You can withdraw your consent at any time. [Learn more about withdrawing your consent](#withdrawing-your-consent).
+ * **Performance of a Contract.** We may process your personal information when we believe it is necessary to fulfill our contractual obligations to you, including providing our services or at your request prior to entering into a contract with you.
+ * **Vital Interests.** We may process your information where we believe it is necessary to protect your vital interests or the vital interests of a third party, such as situations involving potential threats to the safety of any person.
+
+
+### 4. How Long Do We Keep Your Information?
+
+**In Short:**
+We keep your information for as long as necessary to fulfill the purposes outlined in this privacy notice unless otherwise required by law.
+
+We will only keep your personal information for as long as it is necessary for the purposes set out in this privacy notice, unless a longer retention period is required or permitted by law (such as tax, accounting, or other legal requirements). No purpose in this notice will require us to keep your personal information for longer than eighteen (18) months past the termination of the user's account.
```

Comment:
We cannot guarantee this - the Google Drive is the reason this action stalled some time ago, and is still an unresolved issue. Work needs to be done to put procedures in place before this can reasonably be stated.